docs/api/browser-window.md

```diff
@@ -1,1532 +1,1543 @@
-# BrowserWindow
```

Comment:
A 3000 line diff in the docs...? :smile: 

Please fix these side-effects

Reply:
Ugh, will do, must be whitespace. Thanks!

Reply:
@ckerr fixed